lib/ebpd_loader.h

```diff
@@ -0,0 +1,14 @@
+#ifndef __EBPD_LOADER_H_
+#define __EBPD_LOADER_H_
+
+class EbpdLoader {
+    public:
+        EbpdLoader() { }
+        ~EbpdLoader();
+        int LoadFile(const char *filepath, int ifindex);
```

Comment:
LoadFile should be named in a way that documents that it only works for XDP programs (this is why ifindex is required)

Reply:
Changed the filename to xdp_loader.h/xdp_loader.cc to make sure it is explicit. Please see the new pull request.